onlyswaps-verifier/src/main.rs

```diff
@@ -119,23 +123,26 @@ async fn run_onlyswaps(app_config: &AppConfig) -> anyhow::Result<()> {
             request_id = verification.request_id.to_string(),
             "attempting verification"
         );
-        match signer.evaluate_and_send(&verification).await {
-            Err(e) => {
-                tracing::error!(
-                    chain_id = verification.chain_id,
-                    request_id = verification.request_id.to_string(),
-                    error = e.to_string(),
-                    "verification returned an error"
-                );
+        let signer = signer.clone();
+        tokio::spawn(async move {
+            match signer.evaluate_and_send(&verification).await {
```

Comment:
I guess we could use a semaphore to limit the number of spawned tasks, maybe we can add a comment for future reference